packages/libs/eda/src/lib/core/components/computations/plugins/differentialabundance.tsx

```diff
@@ -94,7 +94,13 @@ export const plugin: ComputationPlugin = {
     volcanoplot: volcanoPlotVisualization.withOptions({
       getPlotSubtitle(config) {
         if (DifferentialAbundanceConfig.is(config)) {
-          return `Differential abundance computed using ${config.differentialAbundanceMethod} with default parameters.`;
+          return `Differential abundance computed using ${
```

Comment:
This would be regular JSX too, e.g.
```
return <span>Differential abundance computed using {method} {CITATIONS[method]} with default parameters</span>;
```